src/common/components/repositories-home/index.js

```diff
@@ -109,3 +123,4 @@ function mapStateToProps(state) {
 }
 
 export default connect(mapStateToProps)(withRouter(RepositoriesHome));
+export const raw = RepositoriesHome;
```

Comment:
I know we didn't settle down on any convention for this, but is basically the same as exporting `RepositoriesHome` directly - maybe just under different name, like `RepositoriesHomeView`.

But `RepositoriesHomeRaw` works for me as well. Simply - I would be for exporting it under full uppercase class name. It doesn't matter much in this case with proxyquire, but it does with import.